### PR TITLE
Remove support for cached zip builds.

### PR DESCRIPTION
**Pull request by ryanwilson**
This never really worked that well, and isn't used since we're now
building the zip file on Kokoro. Quick iterations can be handled with
changing the `podsToInstall` variable leaving this code outdated and
unnecessary.

### CODE REVIEW

**Comment by ryanwilson** [comment]
Will need to build a full candidate to ensure this is fine, but I believe this should be a no-op.

**Comment by ryanwilson** [comment]
The force push was rebasing onto master, this succeeds building properly.

File: ZipBuilder/Sources/ZipBuilder/main.swift [hunk 1]
```diff
@@ -16,19 +16,17 @@
 
 import Foundation
 
+// Delete the cache directory, if it exists.
+do {
+  let cacheDir = try FileManager.default.firebaseCacheDirectory()
```
Comment (paulb777) [inline ZipBuilder/Sources/ZipBuilder/main.swift]: Looks like cache was overloaded before - and now just means one thing?

Comment (ryanwilson) [reply ZipBuilder/Sources/ZipBuilder/main.swift]: Yes, was overloaded before. the cache directory here is more of a temporary scratch folder for us to write to and is not used between builds. Perhaps there's a better name for it?

**Pull request merged**